tests/testsuite/update.rs

```diff
@@ -661,15 +650,14 @@ fn update_recursive_conflicts_with_precise() {
 
     p.cargo("update serde:0.2.1 --precise 0.2.2 --recursive")
         .with_status(1)
-        .with_stderr(
-            "\
-error: the argument '--precise <PRECISE>' cannot be used with '--recursive'
+        .with_stderr_data(str![[r#"
+[ERROR] the argument '--precise <PRECISE>' cannot be used with '--recursive'
 
 Usage: cargo[EXE] update --precise <PRECISE> <SPEC|--package [<SPEC>]>
```

Comment:
Consider adding an auto-redaction for `cargo[EXE]`.

Reply:
Could you expand on why?

If we could redact `cargo` and `cargo.exe` to a `[CARGO_BIN]` that would remove the need for Linux users to add `[EXE]` at the end when they see CI fail.  However, I don't think we can definitively say the bare word `cargo` should map to `CARGO_BIN`.

Reply:
Based on my experience implementing recent PRs, my workflow has been to simply replace the content of deprecated functions and rely on the tests to overwrite them. I then compare the overwritten versions with the originals and make necessary modifications.

However, currently, binary paths on Darwin/Linux platforms lack the `[EXE]` extension, and you need to add it manually.

I am also fine with appending `[EXE]` to the binary (including `bin:`, `[EXECUTABLE]` , other binaries in `../bin/target`, etc.) which should significantly improve the user experience.

Reply:
Ok, so it was to try to find a way to avoid manually adding `[EXE]`.  I wish there was a way to automate this but I cannot think of one.